### PR TITLE
adjust to Fairway's package and group/artifact naming conventions

### PR DESCRIPTION
**Pull request by malaffoon**
Yeah, lot of files but mostly brainless stuff ...

### CODE REVIEW

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/model/Exam.java [hunk 1]
```diff
@@ -17,7 +19,7 @@ public Integer getAchievementLevel() {
     }
 
     public List<ExamClaim> getExamClaims() {
-        return examClaims == null ? newArrayList() : newArrayList(examClaims);
+        return examClaims == null ? newArrayList() : Lists.newArrayList(examClaims);
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/model/Exam.java]: Hmm, not intentional, i'll revert this

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/DistrictRepository.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package org.opentestsystem.rdw.ingest.processor.repository;
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/DistrictRepository.java]: Hmm. @agorina should this file exist?

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/AnyExamProcessor.java [hunk 3]
```diff
@@ -153,8 +151,8 @@ private Map parseExaminee(final Examinee examinee) throws ParseException {
         return studentData;
     }
 
-    private void parseExamineeAttribute(ExamineeAttribute attribute, Student.Builder studentBuilder, StudentAttributes.Builder studentAttributesBuilder) throws ParseException {
-        if (attribute.getContext().value() != "FINAL") return;
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/AnyExamProcessor.java]: i noticed a bunch of these warnings; i fixed a few and then decided to clean them up later as part of another checkin

Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/service/AnyExamProcessor.java]: I am planning on fixing it all. There is clean and refactoring that I am planning on doing next.

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/DistrictRepository.java (hunk 2 above)
Comment (agorina) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/DistrictRepository.java]: If you are asking if DistrictRepository.java should exists, then the answer is yes. Not sure if this was the question. Let me know if it is not.

**Pull request merged**